bundle/build

```diff
@@ -85,9 +90,40 @@ if [[ $ERRORED == 1 ]]; then
     exit 1
 fi
 
+# Local assets
 cp Makefile $TMPDIR
 cp README.md $TMPDIR
 
+curl_to() {
+    curl -sfL -o "$1" "$2"
+}
+TMP_BIN=$TMPDIR/bin
+
+# conmon
+curl_to $TMP_BIN/conmon \
+    https://github.com/containers/conmon/releases/download/$CONMON_VERSION/conmon
```

Comment:
For security reasons, I would check the sha256sum from each file from curl.

Reply:
@dougsland Do you mean to hardcode the checksum next to version and check against it?

Something like
```shell
CONMON_VERSION=v2.0.11
CONMON_CHECKSUM=xxxxxx
....
curl_to() {
￼    curl -sfL -o "$1" "$2"
    if ! -z $3 -a ! sha512sum "$1" | grep -F "^$3"; then
        echo 2>&1 "Bad checksum for $2"
        exit 1
    fi
}
...

curl_to $TMP_BIN/conmon $CONMON_URL $CONMON_CHECKSUM
￼    https://github.com/containers/conmon/releases/download/$CONMON_VERSION/conmon
```

I mean, this will work, but it would be harder to bump versions

Reply:
@kolyshkin Not really, I thinking something like the pseudo-code bellow. However, I don't see in the assets file a conmon-$release.sha256sum available in here: https://github.com/containers/conmon/releases/

@haircommander might be a good idea to have a sha256sum file in case we don't have for validation. As example, runc provides it, see: https://github.com/opencontainers/runc/releases


  tmpdir_wget=$(mktemp -d)
  pushd "${tmpdir_wget}" &> /dev/null || exit

  wget conmon
  wget conmon.sha256sum

  sha256sum_from_tarfile=$(sha256sum conmon-$version.tar | awk '{print $1}')
  sha256sum_from_web=$(cat conmon-sha256-file)

    if [ "${sha256sum_from_tarfile}" != "${sha256sum_from_web}" ]; then
        fail....
        exit 1
    fi
    echo "passed..." 
   
  popd &> /dev/null || exit
  ${RM_BIN} -rf "${tmpdir_wget}"

Reply:
Hm, I'd like to follow-up on that in another PR. Is that okay for you?